app/views/idv/welcome/show.html.erb

```diff
@@ -0,0 +1,123 @@
+<% title t('doc_auth.headings.welcome') %>
+
+<% content_for(:pre_flash_content) do %>
+  <%= render StepIndicatorComponent.new(
+        steps: Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS,
+        current_step: :getting_started,
+        locale_scope: 'idv',
+        class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
+      ) %>
+<% end %>
+
+<%= render 'shared/maintenance_window_alert' do %>
+  <%= render JavascriptRequiredComponent.new(
+        header: t('idv.welcome.no_js_header'),
+        intro: t('idv.welcome.no_js_intro', sp_name: decorated_session.sp_name || t('doc_auth.info.no_sp_name')),
+      ) do %>
+
+    <% if @current_user&.reproof_for_irs?(service_provider: @current_sp) %>
+    <%= render AlertComponent.new(
+          type: :info,
+          message: t('doc_auth.info.irs_reproofing_explanation'),
+          class: ['margin-bottom-2', 'usa-alert--info-important'],
+        )
+    %>
+    <% end %>
+
+  <%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.welcome')) %>
+    <p>
+      <%= t('doc_auth.info.welcome_html', sp_name: decorated_session.sp_name || t('doc_auth.info.no_sp_name')) %>
+    </p>
+
+    <h2><%= t('doc_auth.instructions.welcome') %></h2>
+
+    <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
+      <%= c.with_item(heading: t('doc_auth.instructions.bullet1')) do %>
+        <p><%= t('doc_auth.instructions.text1') %></p>
+      <% end %>
+      <%= c.with_item(heading: t('doc_auth.instructions.bullet2')) do %>
+        <p><%= t('doc_auth.instructions.text2') %></p>
+      <% end %>
+      <%= c.with_item(heading: t('doc_auth.instructions.bullet3')) do %>
+        <ul class="usa-list">
+          <% t('doc_auth.instructions.text3_html').each do |bullet_item| %>
+            <li><%= bullet_item %></li>
+          <% end %>
+        </ul>
+        <%= new_tab_link_to(
+              t('idv.troubleshooting.options.learn_more_address_verification_options'),
+              help_center_redirect_path(
+                category: 'verify-your-identity',
+                article: 'phone-number',
+                flow: :idv,
+                step: :welcome,
+                location: 'you_will_need',
+              ),
+            ) %>
+      <% end %>
+    <% end %>
+
+    <%= simple_form_for :doc_auth,
+                        url: url_for,
+                        method: 'put',
+                        html: { autocomplete: 'off', class: 'margin-y-5 js-consent-continue-form' } do |f| %>
+      <%= f.submit t('doc_auth.buttons.continue') %>
```

Comment:
I know we are keeping feature parity with the FSM so this is probably fine for now, but ultimately this should probably just be a link to the agreement step, right?

Reply:
Good point! Could even do that as part of the delete PR for agreement step.